src/libs/OptionsListUtils.js

```diff
@@ -215,7 +215,7 @@ function createOption(personalDetailList, report, draftComments, {
         login: !hasMultipleParticipants ? personalDetail.login : null,
         reportID: report ? report.reportID : null,
         isUnread: report ? report.unreadActionCount > 0 : null,
-        hasDraftComment: reportDraftComment && reportDraftComment.length > 0,
+        hasDraftComment: !!reportDraftComment && reportDraftComment.length > 0,
```

Comment:
I like `!!` but it would be better if this were a direct `Boolean` casting for clarity. 
```suggestion
        hasDraftComment: Boolean(reportDraftComment) && reportDraftComment.length > 0,
```

Reply:
Another possible solution is `_.size(reportDraftComment) > 0` 
_.size should be safe to handle empty, null or undefined.